src/hotspot/cpu/x86/stubGenerator_x86_32.cpp

```diff
@@ -3506,9 +3506,9 @@ class StubGenerator: public StubCodeGenerator {
     const Register d = rbx;
     const Register g = rsi;
     const Register h = rdi;
-    const Register empty = 0; // will never be used, in order not
-                              // to change a signature for crc32c_IPL_Alg2_Alt2
-                              // between 64/32 I'm just keeping it here
+    const Register empty = noreg; // will never be used, in order not
```

Comment:
Shade and I were looking at this bug a week or so ago, but agreed to leave it because this patch would find and fix it. Good!